lib/associations/has-one.js

```diff
@@ -18,7 +18,7 @@ const Association = require('./base');
  */
 class HasOne extends Association {
   constructor(srcModel, targetModel, options) {
```

Comment:
Should name it source, target like all other associations, Please change the names